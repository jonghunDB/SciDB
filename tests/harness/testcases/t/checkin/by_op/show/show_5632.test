--setup
--start-query-logging
--test

# The query that crashed the engine.
show('build(<x:datetimetz> [i=0:0,1,0], null)', 'afl')

# The other queries that were mentioned in SDB-5632.
show('build(<x: datetime> [i=0:0,1,0], null)', 'afl')
show('build(<x:float> [i=0:0,1,0], null)', 'afl')
show('build(<x:int64> [i=0:0,1,0], null)', 'afl')

--cleanup
--stop-query-logging
