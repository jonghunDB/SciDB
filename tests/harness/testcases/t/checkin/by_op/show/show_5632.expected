SCIDB QUERY : <show('build(<x:datetimetz> [i=0:0,1,0], null)', 'afl')>
{i} schema
{0} 'build<x:datetimetz> [i=0:0:0:1]'

SCIDB QUERY : <show('build(<x: datetime> [i=0:0,1,0], null)', 'afl')>
{i} schema
{0} 'build<x:datetime> [i=0:0:0:1]'

SCIDB QUERY : <show('build(<x:float> [i=0:0,1,0], null)', 'afl')>
{i} schema
{0} 'build<x:float> [i=0:0:0:1]'

SCIDB QUERY : <show('build(<x:int64> [i=0:0,1,0], null)', 'afl')>
{i} schema
{0} 'build<x:int64> [i=0:0:0:1]'

